src/controllers/user.controller/fetchUser/queries/queries.js

```diff
@@ -86,7 +86,8 @@ query getUser($id: Int, $cognito_sub: String) {
       _and: [{user: { cognito_sub: { _eq: $cognito_sub }}}, {user2: { _eq: $id }}]
     }
   ]}) {
-      status
+      status,
+      user1
```

Comment:
You can alias fields in the query itself as`sender_id: user1`. Check https://hasura.io/blog/customising-alias-graphql-fields-with-hasura/